DigitalOffice/Services/ProjectService/Controllers/ProjectController.cs

```diff
@@ -30,5 +30,13 @@ public class ProjectController : ControllerBase
         {
             return command.Execute(projectId, request);
         }
+
+        [HttpDelete("disableWorkersInProject")]
+        public void DisableWorkersInProject(
+            [FromServices] IDisableWorkersInProjectCommand command,
+            [FromQuery] WorkersIdsInProjectRequest request)
+        {
+            command.Execute(request);
```

Comment:
I agree that the command is void. But shouldn't we response with at least something like a Status Code? I think this question should be addressed on the next call. I'm afraid I won't be able to do that, so please keep that in mind.